docs/whats-new.asciidoc

```diff
@@ -13,75 +13,71 @@ Other versions: {security-guide-all}/8.5/whats-new.html[8.5] | {security-guide-a
-
-[role="screenshot"]
-image::whats-new/images/8.6/DED-integration.png[DED integration]
+A new `get file` {security-guide}/response-actions.html[response console] action allows you to retrieve and download a file from a host. Files are downloaded in a password-protected `.zip` archive to prevent the file from running. 
 
 [float]
-== Entity Analytics dashboard enhancements
-
-The following enhancements have been made to the {security-guide}/detection-entity-dashboard.html[Entity Analytics dashboard] to improve efficiency in responding to emerging threats: 
+== Data Quality dashboard 
 
-* Hover over a host name or user name to display inline actions: *Add to timeline investigation*, which adds the selected value to a new Timeline, and *Copy to Clipboard*, which copies the host name value for you to paste later.
-* Launch Timeline by clicking the number link in the *Alerts* column. 
-* Install and run a {ml} job directly from the *Notable Anomalies* table  by clicking *Run job*. 
+The {security-guide}/data-quality-dash.html[Data Quality] dashboard allows you to quickly check one or many indices for unmapped fields or fields with mapping conflicts, and makes it easy to track and share the results. 
 
 [role="screenshot"]
-image::whats-new/images/8.6/entity-dashboard.png[Entity Analytics Dashboard]
+image::whats-new/images/8.7/data-qual-dash.png[Data Quality dashboard]
 
 [float]
 == Exceptions enhancements
 
-The following enhancements improve the exceptions feature's UI and simplifies management: 
+There are several enhancements and UI improvements to rule exceptions:
 
-* The Exceptions lists page has been renamed to *Shared Exception Lists*, where you can create, manage, and import shared exception lists.
-* There are now two types of exception lists:
-** *Default rule list*: A default rule list is a group of exceptions that belong to a single rule. Other rules cannot use exceptions within a rule’s default rule list.
-+
-[role="screenshot"]
-image::whats-new/images/8.6/default-rule-list.png[Default rule list]
-+
-** *Shared exception list*: A shared exception list allows you to group exceptions together, then apply them to multiple rules. Exception lists created in {stack} versions 8.5 and earlier become shared exception lists in 8.6 or later. You can access all shared exception lists from the *Shared Exception Lists* page (*Manage -> Shared Exception Lists*).
-+
-[role="screenshot"]
-image::whats-new/images/8.6/rule-exceptions-page.png[Shared Exceptions list]
-* You can now duplicate rules and their exceptions, or rules only.
+* When {security-guide}/detections-ui-exceptions.html#detection-rule-exceptions[adding a rule exception], you can set an expiration date for the exception so it only temporarily applies. You can view active and expired exceptions under the *Rule exceptions* tab and choose whether to include expired exceptions when exporting rules.
```

Comment:
```suggestion
* When {security-guide}/add-exceptions.html#detection-rule-exceptions[adding a rule exception], you can set an expiration date for the exception so it only temporarily applies. You can view active and expired exceptions under the *Rule exceptions* tab and choose whether to include expired exceptions when exporting rules.
```

Reply:
applied in https://github.com/elastic/security-docs/pull/3111/commits/bb1604b802247c73626d7f4cb19872fc5ba12de9